src/utils/indent-string.ts

```diff
@@ -0,0 +1,3 @@
+export function indentString (string: string, count: number): string {
+  return string.replace(/^(?!\s*$)/gm, ' '.repeat(count))
+}
```

Comment:
This file doesn't have a newline at the end of it... is this a big problem @cenk1cenk2?

Reply:
Fixed e04c796